tests/test_cc_visa.py

```diff
@@ -109,17 +110,15 @@ def test_cc_close(constructor_params_serial):
     assert visa_inst._cc_close() is None
 
 
-@pytest.mark.parametrize(
-    "constructor_params_serial, raw_value, expected_return",
-    [
-        (constructor_params_serial, True, {"msg": b"read response"}),
-        (constructor_params_serial, False, {"msg": "read response"}),
-    ],
-)
-def test__cc_receive(mocker, constructor_params_serial, raw_value, expected_return):
+def test_cc_receive(
+    mocker,
+):
     visa_inst = cc_visa.VISASerial(constructor_params_serial)
```

Comment:
What does it contain now?

Reply:
constructor_params_serial is a dict defined at the start of the file. https://github.com/eclipse/kiso-testing/blob/master/tests/test_cc_visa.py#L16